R/utils.R

```diff
@@ -144,29 +152,25 @@ app_path <- function(path, arg = "path") {
     stop(paste0("'", path, "' doesn't exist"), call. = FALSE)
   }
 
-  if (is_app(path)) {
-    app <- path
-    dir <- path
-  } else if (is_rmd(path)) {
+  if (is_rmd(path)) {
```

Comment:
Previously it checked if it was an app, then an Rmd, but now the oder of those checks is reversed. I think this could cause breakage for app directories that contain both an app.R/server.R and an .Rmd. For example: https://github.com/rstudio/shinycoreci-apps/tree/master/apps/016-knitr-pdf